web/app.js

```diff
@@ -2046,6 +2046,69 @@ function handleMouseWheel(evt) {
 window.addEventListener('DOMMouseScroll', handleMouseWheel);
 window.addEventListener('mousewheel', handleMouseWheel);
 
+function touchSwipeDetector(evt) {
```

Comment:
The standard in PDF.js is to squash fixes of review comments into the base commit, so if you could do that it would be great!